Deleting/editing of bookmarks fails

Hi there,

I am working on a pretty old thinkpad (64bit arch linux, up to date) and I have a few issues. Opening buku-run and seeing/filtering/opening of bookmarks works fine but editing and deleting them appears broken.
```
open buku-run
select bookmark
Alt+a
select delete
confirm
```
appears to exit correctly but when opening buku-run again the bookmark is still there and can be opened with the browser, repeating the procedure on the same bookmark prints 

`[ERROR] Incorrect index or range`

Navigating to the bookmark editing dialog an hitting enter on either `url` or `tags` just exits doing nothing.

What am I doing wrong :D?

As a side note, the [arch user repository package](https://aur.archlinux.org/packages/buku_run-git/) is broken and I am unfamiliar with the aur website. I don't really know where to report the issue, the installation fails with
```
==> Starting package()...
install: cannot stat 'buku': No such file or directory
==> ERROR: A failure occurred in package().
    Aborting...
```
 
so I manually installed the deps, cloned the repo and run the binary directly from its directory as the make install does not appear to perform any magic.

--- a/buku_run
+++ b/buku_run
@@ -130,7 +130,7 @@ optionsMenu () {
 }
 
 deleteMenu () {
-    id=$(echo "${menu}" | awk -F '\t' '{ print $1 }')
+    id=$(echo "${menu}" | awk '{ print $1 }')
     delask=$(echo -e "1. Yes\n2. No" | _rofi -p '> ' -mesg "Really delete bookmark?")
     val=$?
     if [[ $val -eq 1 ]]; then
@@ -146,9 +146,9 @@ deleteMenu () {
 }
 
 editMenu () {
-    bookmark=$(echo "${menu}" | awk -F '\t' '{ print $2 }')
-    id=$(echo "${menu}" | awk -F '\t' '{ print $1 }')
-    tags=$(echo "${menu}" | awk -F "\t" '{ print $3 }')
+    bookmark=$(echo "${menu}" | awk '{ print $2 }')
+    id=$(echo "${menu}" | awk '{ print $1 }')
+    tags=$(echo "${menu}" | awk '{ print $3 }')
     content=$(echo -e "url:  $bookmark\ntags: $tags")
     editmenu=$(echo -e "< Return\n---\n${content}" | _rofi -p '> ')
     val=$?

